{
    "title": "Openpyxl 2.4.1 resolves external entities by default, which allows remote attackers to conduct XXE attacks via a crafted .xlsx document.",
    "score": 8.2,
    "severity": "HIGH",
    "cwe_id": "CWE-611",
    "cvss_vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:N/A:H",
    "cpe_match": [
        "cpe:2.3:a:python:openpyxl:2.4.1:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": []
}
